RealmSwift.podspec

```diff
@@ -0,0 +1,31 @@
+Pod::Spec.new do |s|
+  s.name                    = 'RealmSwift'
+  s.version                 = `sh build.sh get-version`
+  s.summary                 = 'Realm is a modern data framework & database for iOS & OSX.'
+  s.description             = <<-DESC
+                              Realm is a modern data framework & database for iOS & OSX. You can use it purely in memory — or persist to disk with extraordinary performance.
+
+                              Realm’s data structures look like NSObjects and NSArrays, but provide additional features such as: querying, relationships & graphs, thread safety, and more.
```

Comment:
Take 2, much shorter:

**Realm.podspec**

The Realm database, for Objective-C. (If you want to use Realm from Swift, see the “RealmSwift” pod.)

Realm is a mobile database: a replacement for Core Data & SQLite. You can use it on iOS & OSX. Realm is not an ORM on top SQLite: instead it uses its own persistence engine, built for simplicity (& speed). Learn more and get help at https://realm.io

**RealmSwift.podspec**

The Realm database, for Swift. (If you want to use Realm from Objective-C, see the “Realm” pod.)

Realm is a mobile database: a replacement for Core Data & SQLite. You can use it on iOS & OSX. Realm is not an ORM on top SQLite: instead it uses its own persistence engine, built for simplicity (& speed). Learn more and get help at https://realm.io